client/src/pages/Contacts/index.js

```diff
@@ -1,4 +1,173 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Axios from 'axios';
 
-const Contacts = () => <div>Hi from Contacts</div>;
+import {
+  Divider,
+  Typography,
+  Container,
+  TextareaAutosize,
+} from '@material-ui/core';
+import Snackbar from '@material-ui/core/Snackbar';
+import { useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import Alert from '@material-ui/lab/Alert';
+
+import {
+  Header,
+  Footer,
+  InputField,
+  ButtonComponent,
+  Loading,
+} from '../../component';
+import { contactValidationSchema } from '../../utils';
+import map from '../../assets/images/map.png';
+import useStyles from './style';
+
+const Contacts = () => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('xs'));
+
+  const [username, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [mobile, setMobile] = useState('');
+  const [message, setMessage] = useState('');
+  const [error, setError] = useState();
+  const [loading, setLoading] = useState(false);
+  const [open, setOpen] = useState(false);
+  const classes = useStyles();
+  const handleClose = (reason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setOpen(false);
+  };
+
+  const clear = () => {
+    setMessage('');
+    setEmail('');
+    setMobile('');
+    setName('');
+    setError(null);
+  };
+
+  const handelChange = ({ target: { value, name } }) => {
+    switch (name) {
+      case 'username':
+        setName(value);
+        break;
+      case 'email':
+        setEmail(value);
+        break;
+      case 'mobile':
+        setMobile(value);
+        break;
+      case 'message':
+        setMessage(value);
+        break;
+      default:
+        break;
+    }
+  };
+  const handleSubmit = async (e) => {
+    try {
+      e.preventDefault();
+      setLoading(true);
+      const contactInfo = { username, email, message, mobile };
+      await contactValidationSchema.isValid(contactInfo, { abortEarly: false });
+      await Axios.post('/api/v1/contact-us', contactInfo);
+      clear();
+      setLoading(false);
+      setOpen(true);
+    } catch (err) {
+      setError(
+        err.response.data.message
+          ? err.response.data.message
+          : 'Internal server error'
```

Comment:
I think this message make user panic, change it to something less friendly like `something went wrong, please try again later